src/runtime_src/core/tools/xbutil2/SubCmdFan.cpp

```diff
@@ -27,23 +27,33 @@ namespace po = boost::program_options;
 // System - Include Files
 #include <iostream>
 
+// ======= R E G I S T E R   T H E   S U B C O M M A N D ======================
+#include "SubCmd.h"
+static const unsigned int registerResult = 
+                    register_subcommand("fan", 
```

Comment:
just curious, we are able to set fan speed from command line?

Reply:
The "fan" and "run" subcommands were added prior to examining to see if they are actually implemented.  I will remove them from consideration.